tests/modules/nf-core/ashlar/main.nf

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ASHLAR } from '../../../../../modules/modules/nf-core/ashlar/main.nf'
+// we zero out the UUID of output tiff images with ZERO_UUID so we get a consistent md5sum
+include { ZERO_UUID } from './zero_uuid.nf'
+
+def TEST_ORIG_IMG = "/home/pollen/HITS/nextflow/mcmicro/exemplar-001/raw/exemplar-001-cycle-0{6,7}.ome.tiff"
+def TEST_ORIG_IMG_1 = "/home/pollen/HITS/nextflow/mcmicro/exemplar-001/raw/exemplar-001-cycle-06.ome.tiff"
+def TEST_ORIG_IMG_2 = "/home/pollen/HITS/nextflow/mcmicro/exemplar-001/raw/exemplar-001-cycle-07.ome.tiff"
+def TEST_ORIG_DFP_1 = "/home/pollen/HITS/nextflow/mcmicro/exemplar-001/illumination/exemplar-001-cycle-06-dfp.tif"
+def TEST_ORIG_FFP_1 = "/home/pollen/HITS/nextflow/mcmicro/exemplar-001/illumination/exemplar-001-cycle-06-ffp.tif"
+
+def TEST_IMG_ALL = "/home/pollen/HITS/nextflow/mcmicro/cycif-tonsil/cycif-tonsil-cycle{1,2,3}.ome.tif"
+def TEST_IMG_1 = "/home/pollen/HITS/nextflow/mcmicro/cycif-tonsil/cycif-tonsil-cycle1.ome.tif"
+def TEST_IMG_2 = "/home/pollen/HITS/nextflow/mcmicro/cycif-tonsil/cycif-tonsil-cycle2.ome.tif"
+def TEST_IMG_3 = "/home/pollen/HITS/nextflow/mcmicro/cycif-tonsil/cycif-tonsil-cycle3.ome.tif"
+
+workflow test_ashlar_one_file {
+
+    input_list =  [ [ [ id:'test_1', args: '--flip-y' ],
```

Comment:
We can't use --flip-y and --flip-x when the input image doesn't need it. The output will be nonsensical.

Reply:
Ok, sounds good.  I was just looking for something simple to test with.
I can just do a test with no args and the 3 cycif tonsil images, but I'd also like to have a test of the args (I see above that we'll need to change how we handle them though).
Would using --filter-sigma or --tile-size make sense as a test parameter?  If so, what's a reasonable value to use?

Reply:
Sounds good.  Thanks!
I've added a tile-size test.